state/migration_export.go

```diff
@@ -918,10 +918,11 @@ func (e *exporter) addApplication(ctx addApplicationContext) error {
 		if cloudContainer, found := ctx.cloudContainers[unit.globalKey()]; found {
 			args.CloudContainer = e.cloudContainer(cloudContainer)
 		}
-		if args.State, err = unit.State(); err != nil {
+		// TODO: hml 18-mar-2020
+		// add the rest of unit.State() to model migration
+		if args.State, _, err = unit.State(); err != nil {
```

Comment:
Same comment about minimizing TODOs. I would recommend updating juju/description with the required fields and then updating this PR to use them.